static/app/views/issueDetails/groupActivity.spec.jsx

```diff
@@ -338,4 +340,63 @@ describe('GroupActivity', function () {
       expect(deleteMock).toHaveBeenCalledTimes(1);
     });
   });
+
+  it('renders ignored', function () {
+    createWrapper({
+      activity: [
+        {
+          id: '123',
+          type: GroupActivityType.SET_IGNORED,
+          data: {
+            ignoreUntilEscalating: true,
+          },
+          user: TestStubs.User(),
+          dateCreated,
+        },
+      ],
+    });
+    expect(screen.getAllByTestId('activity-item').at(-1)).toHaveTextContent(
+      'Foo Bar ignored this issue'
+    );
+  });
+
+  it('renders ignored until escalating', function () {
+    createWrapper({
+      activity: [
+        {
+          id: '123',
+          type: GroupActivityType.SET_IGNORED,
+          data: {
+            ignoreUntilEscalating: true,
+          },
+          user: TestStubs.User(),
+          dateCreated,
+        },
+      ],
+      organization: {features: ['escalating-issues-ui']},
+    });
+    expect(screen.getAllByTestId('activity-item').at(-1)).toHaveTextContent(
+      'Foo Bar archived this issue until it escalates'
+    );
+  });
+
+  it('renders ignored until it happens x times in time window', function () {
```

Comment:
Is there a reason, out of all of the options other than archived-until-escalating, you're testing this one? Maybe we should test them all?